CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Changed
 
  - upgrade django-machina from 1.1.3 to 1.1.4
+ - update readme instructions on accessing backoffice 
```

Comment:
IMO there is no need to update the changelog for this kind of modification.

Reply:
How do we deal with circle ci rule then ?

Reply:
The job `check-changelog` success is not required to merge the PR.

Reply:
Exact and regarding the last commit it is not needed, I'll remove it.